### ISSUE TITLE
Elixir 1.8 warning from ABCSize check

### ISSUE THREAD

**Issue by lucas-nelson** [rrrene/credo#625]
### Environment

* Credo version (`mix credo -v`): `1.0.0`
* Erlang/Elixir version (`elixir -v`): `1.8.0`
* Operating system: MacOS 10.14.2 & Alpine 3.8 (docker container)

### What were you trying to do?

`mix credo`

### Expected outcome

No warnings from credo code.

### Actual outcome

```
warning: the Collectable protocol is deprecated for non-empty lists. The behaviour of things like Enum.into/2 or "for" comprehensions with an :into option is incorrect when collecting into non-empty lists. If you're collecting into a non-empty keyword list, consider using Keyword.merge/2 instead. If you're collecting into a non-empty list, consider concatenating the two lists with the ++ operator.
  (elixir) lib/collectable.ex:83: Collectable.List.into/1
  (elixir) lib/enum.ex:1227: Enum.into_protocol/2
  lib/credo/check/refactor/abc_size.ex:152: Credo.Check.Refactor.ABCSize.traverse_abc/3
  (elixir) lib/macro.ex:286: anonymous fn/4 in Macro.do_traverse_args/4
  (elixir) lib/enum.ex:1431: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
```

**Comment by lucas-nelson** [comment]
I can the same warning if I clone master, remove `.tool-versions` (so it will pick up the latest versions in my global config), and `mix test`.

(and one failed test - `test it should NOT report expected code (Credo.Check.Refactor.PipeChainStartTest)`)

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by asummers** [comment]
This is now fixed in `1.0.1`
